packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -2439,12 +2550,30 @@ ComposableTableBasic = () => {
 
 ### Composable: Row click handler, custom row wrapper, header tooltips & popovers
 
-This example demonstrates customizing rows, adding tooltip and popover information to header items, and some other misc. props.
-
 - If you add the `noWrap` prop to `Thead`, it won't wrap it if there is no space
 - You can add the `textCenter` prop to `Th` or `Td` to center the contents
 - You can pass `onClick`, `className`, `style` and more to `Tr`
 
+To add a header tooltip or popover to `Th`, pass a `ThInfoType` object via the `info` prop.
+
+`ThInfoType` (excerpt): 
+```
+interface ThInfoType {
```

Comment:
I don't know that these excerpts are needed once you add the interfaces to the MD file.  Maybe a link to the interface docs?